ngraph_bridge/ngraph_mark_for_clustering.h

```diff
@@ -30,14 +30,27 @@ Status MarkForClustering(Graph* graph, std::set<string> skip_these_nodes,
                          const string& current_backend);
 Status IsSupportedByBackend(
     const Node* node, const ngraph::runtime::Backend* op_backend,
-    std::map<std::string, std::set<std::shared_ptr<ngraph::Node>>>&
+    const std::map<std::string, std::set<std::shared_ptr<ngraph::Node>>>&
         TFtoNgraphOpMap,
     bool& is_supported);
 bool NodeIsMarkedForClustering(const Node* node);
 void GetStaticInputs(const Node* node, std::vector<int32>* inputs);
 bool InputIsStatic(const Node* node, int index);
 Status GetNodeBackend(const Node* node, string* backend_name);
 void SetNodeBackend(Node* node, const string& backend_name);
+
+using SetAttributesFunction = std::function<Status(Node*)>;
+const std::map<std::string, SetAttributesFunction>& GetAttributeSetters();
+
+using TypeConstraintMap =
+    std::map<std::string, std::map<std::string, gtl::ArraySlice<DataType>>>;
+const TypeConstraintMap& GetTypeConstraints();
```

Comment:
minor GetTypeConstraints ---> GetTypeConstraintMap since that is what it is returning

Reply:
Done.